CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master (unreleased)
 
+### Changes
+* Change `Naming/UncommunicativeMethodParamName` add `to` to allowed names in default config. ([@unused][])
```

Comment:
Please write the link to this PR at the beginning 😃 

```diff
- * Change `Naming/UncommunicativeMethodParamName` add `to` to allowed names in default config. ([@unused][])
+ * [#5626](https://github.com/bbatsov/rubocop/pull/5626): Change `Naming/UncommunicativeMethodParamName` add `to` to allowed names in default config. ([@unused][])
```

Reply:
sorry and thx :smile: